.djlintrc

```diff
@@ -0,0 +1,18 @@
+{
+  "blank_line_after_tag": "load,extends",
+  "blank_line_before_tag": "load,extends,block, macro",
+  "close_void_tags": true,
+  "ignore": "J018,H006,H021,H023,H029,T002,T003,T028",
```

Comment:
Out of curiosity, what are these codes?

Reply:
Sure, they are listed here: https://www.djlint.com/docs/linter/#rules
Basically they are codes for certain linting rules